test/plugins/wasi_nn/wasi_nn.cpp

```diff
@@ -155,12 +155,30 @@ TEST(WasiNNTest, OpenVINOBackend) {
   EXPECT_TRUE(FuncInst->isHostFunction());
   auto &HostFuncGetOutput =
       dynamic_cast<WasmEdge::Host::WasiNNGetOutput &>(FuncInst->getHostFunc());
+  // Get the function "get_output_single".
```

Comment:
I got confused. Why do you add these tests to the OpenVINO backend? The OpenVINO backend does not implement these LLM-related extension functions. That's why the unit tests will fail when trying to get the non-existing functions.